en/docs/references/extend/rebranding/customizable-login-portal.md

```diff
@@ -0,0 +1,134 @@
+
+### Step 2: Create the basic custom layout resources
+
+To create the basic custom layout resources:
+
+1. On the `authenticationendpoint` folder, navigate to the created `extensions` folder and create `layouts` folder and `customs` folders recursively.
+    
+    ```
+    └── extensions
+        └── layouts
+            └── custom
+    ```
+
+3. Navigate to `webapps/authenticationendpoint/includes/layouts/default` folder, copy the `body.html` file and add it to the created `custom` folder.
+
+4. Create the following files and folder inside the `custom` folder:
+    - `styles.css` file
+    - `script.js` file
+    - `assets` folder
+
+!!! note
+    Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
+
+### Step 3: Add a custom layout design
+
+To add a custom layout to the login page
+
+1. Navigate to `webapps/authenticationendpoint/extensions/layouts/custom/`
+
+    1. Copy the code snippet from the [html file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/body.html) into the `body.html` file.
+
+    2. Copy the code snippet from the [css file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/style.css) into the `styles.css` file.
+
+    3. Copy the code snippet from the [javascript file](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/script.js) into the `script.js` file.
+
+    4. Copy the [illustraction.svg](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/illustration.svg) file into the `assets` folder.
+
+2. Navigate to `webapps/authenticationendpoint/includes` directory and copy the `product-title.jsp` and `product-footer.jsp` files into the `webapps/authenticationendpoint/extensions` directory.
+
+    1. Copy content from the [`jsp` file containing the project title](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/project-title-1.jsp) into the `webapps/authenticationendpoint/extensions/product-title.jsp` file of the app.
+
+    2. Copy content from the [`jsp` file containing the project footer](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-footer-1.jsp) into the `webapps/authenticationendpoint/extensions/product-footer.jsp` file of the app.
+
+
+    !!! note
+        Repeat the above steps for `accountrecoveryendpoint` and `x509certificateauthenticationendpoint` as well.
+        
+        The files used for step 2.a and 2.b are as follow:
+
+        - [Project title](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-title-2.jsp).
+
+        - [Project footer](https://github.com/wso2/docs-is/tree/master/en/docs/assets/code-samples/product-footer-2.jsp)
+
+3. Refresh the browser and check out the added custom layout.
+
+    !!! tip
+        In certain cases, the server might stop working after adding changes. In such a case, restart the server to check the changes.
+
+!!! note 
```

Comment:
Two notes following one after the other.
Why not create another heading `Best practices` for this list.